waiter/integration/waiter/request_timeout_test.clj

```diff
@@ -137,7 +137,8 @@
       (make-request-and-assert-timeout waiter-url request-headers start-time-ms timeout-period-sec)
       (delete-service waiter-url request-headers))))
 
-(deftest ^:parallel ^:integration-slow test-request-queue-timeout-max-instances-limit
+; Marked explicit because it's flaky
```

Comment:
When possible, we like to add some context on the flakiness

Reply:
Hard to exfiltrate from Jenkins :p

Reply:
Maybe just a brief explanation of the nature of the flakiness, if we know it?